css/styles.css

```diff
@@ -0,0 +1,113 @@
+.header-title {
+  color: white;
+  position: relative;
+  bottom: 5px;
+}
+
+.header-bar {
+  background-color: #562b81;
+  font-weight: 400;
+  font-size: 20px;
+  height: 53px;
+}
+
+h1 {
+  font-family: 'Proza Libre', sans-serif;
+  font-weight: 500;
+}
+
+body {
+  background-color: #eee;
+  font-family: 'Proza Libre', sans-serif;
+  font-weight: 600;
+}
+
+#img-wrapper {
+  margin-bottom: 1.75rem;
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 4px;
+}
+
+img {
+  max-width: 100%;
+}
+
+input {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 16px;
+  border-style: solid;
+  border-color: #b6b6b6;
+  border-radius: 0.3rem;
+  border-width: 1px;
+  height: 40px;
+  margin-top: 0.75rem;
+  margin-bottom: 1.75rem;
+  padding-left: 8px;
+  padding-right: 8px;
+}
+
+textarea {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 16px;
+  border-color: #b6b6b6;
+  border-radius: 0.3rem;
+  height: 149px;
+  margin-top: 0.75rem;
+  padding: 8px;
+}
+
+#notes-field-section {
+  display: flex;
+  flex-direction: column;
+}
+
+.title-field {
+  display: flex;
+  flex-direction: column;
+}
+
+.photo-url-field {
+  display: flex;
+  flex-direction: column;
+}
+
+#save-button {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+button {
+  font-size: 16px;
+  background-color: #562b81;
+  border-style: none;
+  border-radius: 0.3rem;
+  color: white;
+  padding: 0.5rem;
+  padding-left: 16px;
+  padding-right: 16px;
+  margin-top: 1.5rem;
+  margin-bottom: 1.5rem;
+}
+
+button:hover {
+  background-color: #875ab4;
+  cursor: pointer;
+}
+
+input:focus {
+  outline: 2px solid #562b81;
+}
+
+textarea:focus {
+  outline: 2px solid #562b81;
+}
+
+@media (width <= 768px) {
+  #img-wrapper {
+    height: 245px;
+  }
+}
```

Comment:
Shouldn't be needed. The screen should handle the resizing of the `height` for us